docs/mutations.md

```diff
@@ -483,6 +624,55 @@ Pt = scipy.linalg.expm(t * rate * Q)[i,j]
 If the top of a branch of length {math}`t` has allele {math}`i`,
 the bottom of the branch has allele {math}`j` with probability `Pt[i,j]`.
 
+
+(sec_mutations_adjusting_for_silent)=
+
+#### Adjusting mutation rates for silent mutations
+
+As discussed in :ref:`sec_mutations_silent`, matrix mutation models can insert *silent* mutations,
+which may make it difficult to set the mutation rate appropriately,
+since by "mutation rate" we mean the rate at which mutations appear that *change* the allelic state.
+However, the "mutation rate" is not in fact a single value: it depends on context,
+as some alleles may tend to mutate more than others.
+
+As an example of how to appropriately adjust the mutation rate to produce a given expected diversity,
+take the following somewhat contrived problem.
+Suppose we generate a tree sequence of 1000 samples of 10Mb from a population of size 1,000,
+which has mutations generated under the HKY model with :math:`\\kappa=0.75`
+and equilibrium frequencies :math:`\\pi = [0.1, 0.2, 0.3, 0.4]`.
+We'd like to choose the mutation rate :math:`\\mu` so that the mean genetic diversity
+is equal to 0.001.
+If this was equal to the usual :math:`\theta = 4Nu`, we'd need a "mutation rate" :math:`u`
+of 2.5e-7, so we know we'll need a number slightly higher than that.
+
+To do this, let's figure out the mean rate of appearance of silent mutations.
+We know that (a) the rate of appearance of silent mutations per nucleotide is given by :math:`\\mu`
+times the diagonal of the transition matrix, and (b) the equilibrium frequency of the four
+nucleotides is given by :math:`\\pi`. Multiplying these together, adding them up, we get
+:math:`2.5 \times 10^{-7} = \\mu (1 - \sum_i \\pi_i P_{ii})`, so
+solving for :math:`\\mu` we get, in this case:
+
+```{code-cell}
+pi = np.array([0.1, 0.2, 0.3, 0.4])
+hky = msprime.HKYMutationModel(kappa=0.75, equilibrium_frequencies=pi)
+P = hky.transition_matrix
+u = 2.5e-7
+mu = u / (1 - np.sum(pi * np.diag(P)))
+print(f"Mutation rate should be increased by a factor of {mu/u:.2f}.")
+```
+
+Now, let's verify we get this nucleotide diversity:
+
+```{code-cell}
+ts = msprime.sim_ancestry(1000, population_size=1000, sequence_length=1e7, recombination_rate=1e-8, random_seed=5)
```

Comment:
Is the recombination rate important here? I'm trying to leave it out where it doesn't matter. I guess if we don't include it here we will get a more skewed estimate because we're only using one tree.

Reply:
Right; that's the idea.